### ISSUE TITLE
[Rails 5.1] Pagination link throws error due to unpermitted parameters

### ISSUE THREAD

**Issue by macowie** [galetahub/ckeditor#746]
"Unable to convert unpermitted parameters to hash"

The next page link attempts to merge the current params with the new page, but ActionController::Parameters now throws exceptions on operations on unpermitted parameters.

**Comment by macowie** [comment]
Is there a reason to merge the existing parameters at all? I was unsure if it had any effect on calling back to the original CKEditor instance, but simply changing the link to the below seems to work fine, I'm still able to embed freshly loaded files

```ruby
ckeditor.attachment_files_path(page: @attachments.next_page)
```

**Comment by macowie** [comment]
Additionally, this fixes #745

**Comment by galetahub** [comment]
Fixed in this commit https://github.com/galetahub/ckeditor/commit/604608bb55b25ae7cf2d791696de6c28bf8c0fa6
